3drepo/3drepo.io
# Issue 3888 - Custom Ticket Comments (with images)

This fixes #3888

#### Description
Comments now support images.

Note a couple of things:
- Images can only be uploaded using the drag n drop system or by using the file icon. Adding them like you would in Markdown (`![image](<url>)`) will render as plain text;
- Editing a comment, should allow to also edit the images; however, the design for this is not ready yet. As such, editing a comment will preserve the existing images.


#### Test cases
- Create screenshots;
- Upload 1 or more images from file system;
- Drag and Drop images (the box should appear as soon as the dragging cursor reaches the new comment area) from file system or from other comments;
- Test the different images combinations (1, 2-5, 6+);
- Reply to a comment with 1 image;
- Reply to a comment with 1+ images;
- Try to expand the images in all the different combinations (including the upload phase)
